### PR TITLE
Add Favirecon GIF

### LINKED ISSUE #3: Add GIF for favirecon
https://github.com/edoardottt/favirecon

Use [`charmbracelet/vhs`](https://github.com/charmbracelet/vhs) to customize the GIF. 

- The GIF must reproduce the command `favirecon -l targets.txt`
- The input file must be a long input file containing different online running services
- Be sure to add here both the .tape and the GIF file.

Let me know if you need some guidance :)

### PR DESCRIPTION
**Pull request by carlospatinos**
I think the work is already in place. Just creating the PR for #hacktoberfest purposes hope you dont mind

### CODE REVIEW

File: favirecon/favirecon.tape [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Display:
+#   Hide                            Hide the subsequent commands from the output
+#   Show                            Show the subsequent commands in the output
+
+Output favirecon.gif
+
+Require echo
+
+Set Shell "bash"
+Set FontSize 32
```
Comment (edoardottt) [inline favirecon/favirecon.tape]: The font is too big, look at the gif

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: favirecon/favirecon.tape (hunk 1 above)
Comment (carlospatinos) [reply favirecon/favirecon.tape]: done

File: favirecon/favirecon.tape [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Require echo
+
+Set Shell "bash"
+Set FontSize 16
+Set Width 1200
+Set Height 600
+
+Type "favirecon -l targets.txt" Sleep 3s  Enter
+
+Sleep 5s
```
Comment (edoardottt) [inline favirecon/favirecon.tape]: Can we wait a lil bit more? Like 8 seconds

**Review by edoardottt — CHANGES_REQUESTED** [review]

File: favirecon/favirecon.tape (hunk 2 above)
Comment (edoardottt) [reply favirecon/favirecon.tape]: Sorry for all these little changes 🙏

Comment (carlospatinos) [reply favirecon/favirecon.tape]: No problem, do you mean instead of 5? (at the end) or instead of 3? on the typing?

Comment (edoardottt) [reply favirecon/favirecon.tape]: instead of 5, at the end. thanks

**Comment by carlospatinos** [comment]
Something funny happened with the gif let me take a look

**Review by edoardottt — APPROVED** [review]
lgtm

**Pull request merged**